Jenkinsfile

```diff
@@ -28,10 +28,16 @@ pipeline {
 
         stage('Build') {
             steps {
-                sh "make build-image build-minimal-assisted-iso-generator-image"
+                sh "skipper make build-image build-minimal-assisted-iso-generator-image"
```

Comment:
it's hard to follow but why sometimes we use skipper and other times we don't?

Reply:
We would like the CI to use the same methodology as anyone else
I'd prefer using only `make` which internally calls `skipper`. 
But while that isn't the case - We would like to be similar as we can.

At the moment we cannot use `skipper` for minikube deployment. But I'd still like to use Dockerfile.assisted-service-build image instead of installing prerequisites on our CI machines